### PR TITLE
Update supported OSes for 1.12

### PR DESCRIPTION
**Pull request by domcleal**
- replace Ubuntu 12.04 with 16.04
- replace Fedora 21 with 24
- remove Debian 7 (Wheezy)
- deprecate EL6

### CODE REVIEW

File: _includes/manuals/1.12/2_quickstart_guide.md [hunk 1]
```diff
@@ -4,12 +4,12 @@ The Foreman installer is a collection of Puppet modules that installs everything
...
 * Debian 8 (Jessie), i386/amd64/armhf
-* Debian 7 (Wheezy), i386/amd64
-* Fedora 21, x86_64
-* Red Hat Enterprise Linux 6 or 7, x86_64
+* Fedora 24, x86_64
+* Red Hat Enterprise Linux 7, x86_64
+* Ubuntu 16.04 (Xenial), i386/amd64/armhf
 * Ubuntu 14.04 (Trusty), i386/amd64/armhf
-* Ubuntu 12.04 (Precise), i386/amd64
+* _Deprecated, but supported: RHEL, CentOS, Scientific Linux or Oracle Linux 6_
```
Comment (mmoll) [inline _includes/manuals/1.12/2_quickstart_guide.md]: I'd prefer "Supported, but deprecated"

**Comment by mmoll** [comment]
A hint like for Debian/jessie "Note that Puppet Labs does not provide Puppet 3.x packages for xenial." would be nice for Ubuntu/xenial

**Comment by mmoll** [comment]
:+1: otherwise

**Comment by domcleal** [comment]
Thanks, both comments addressed.

**Comment by mmoll** [comment]
merged, thanks @domcleal!

**Pull request merged**